### DISCUSSION TITLE
What package should we use?

### DISCUSSION THREAD

**Discussion by olivier-spinelli** [CommunityToolkit/dotnet#52] [Q&A] [answered]
As far as I understand the "Microsoft" prefix was problematic. On NuGet there's 2 packages (I'm interested in the Performance one):
https://www.nuget.org/packages/Microsoft.Toolkit.HighPerformance/
and
https://www.nuget.org/packages/CommunityToolkit.HighPerformance/

Both in 7.1.2 updated the same day... 
Both have a source repository to https://github.com/CommunityToolkit/WindowsCommunityToolkit that is wrong (the actual code is here: https://github.com/CommunityToolkit/dotnet).

Can anyone clarify this?

**Comment by Sergio0694** [comment, marked as answer]
Hi @olivier-spinelli!
We have migrated all code here, we just still need to finish updating the various package info, docs and wikis.
Version 7.1.2 of both packages is exactly the same (for the packages in the .NET Community Toolkit), we published them just to avoid confusion for developers that were wondering about the difference version number between the two. In the future, only the CommunityToolkit.* packages from the .NET Community Toolkit will be updated, and the others will eventually be deprecated and just redirected to these ones. So if you're starting to use them now, definitely grab the CommunityToolkit ones 👍

**NOTE:** just to clarify, the namespace change is **only** done to make things simpler to handle on NuGet, and to better organize across all the various repositories in the CommunityToolkit org. These projects are still first party, published by Microsoft and owned and maintained by Microsoft engineers, and we're also using them internally in several first-party apps (eg. the Microsoft Store) 😊

**Reply by olivier-spinelli** [reply]
Thanks a lot. (And I hope this answer will help others!)
